Update users.js

dockeryz opened the pull request:
Adding the rest of the User methods

dockeryz committed:
Update users.js

Adding the rest of the User methods

dockeryz commented:
need to rename geCurrentUserGuilds to getCurrentUserGuilds

gidsola reviewed (changes requested):
Pending full testing ;)

dockeryz committed:
Fix bad naming, add channel_id

geCurrentUserGuilds -> getCurrentUserGuilds
removing unnecessary parameters in body

dockeryz committed:
Update users.js

dockeryz committed:
typos

fixing typos
usernamem to username
uncomment content-type for leaveGuild
